doc/reference/resource_management/index.rst

```diff
@@ -84,5 +84,20 @@ state.
    :c:type:`onoff_manager`, with only a small reduction in functionality
    (primarily no support for the monitor API).
 
+.. _resource_mgmt_onoff_delayed:
+
+Delayed On-Off Manager
+**********************
+
+A delayed on-off manager is an extension to :ref:`resource_mgmt_onoff` which
+allows scheduled requests to the service. Manager ensures that service will be
```

Comment:
"ensures" is a bit optimistic.  All this can provide is a best effort, and if the user is unaware of the delay requirements transitions may always occur too late.